checkstyle-checks/src/main/resources/sonatype/checkstyle-configuration.xml

```diff
@@ -22,6 +22,12 @@
 
   <module name="SuppressWarningsFilter" />
 
+  <module name="LineLength">
+    <property name="max" value="120"/>
+    <!-- Allow longer lines in comments. -->
+    <property name="ignorePattern" value="^ \*.*"/>
+  </module>
+
```

Comment:
https://checkstyle.org/config_sizes.html#LineLength

LineLength's parent must be Checker, not TreeWalker.